docs/universal-gateway/examples/front-door-pattern.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Implement the 'Front Door' Pattern"
+description: By using Cloud Endpoint and internal Agent Endpoints together, you loosely double your gateway and services for extra flexibility.
+sidebar_label: "The Front Door Pattern"
+---
+
+import ReserveDomain from "./snippets/_reserve-domain.mdx";
+import CloudEndpoint from "./snippets/_cloud-endpoint.mdx";
+import Back from "./snippets/_back-to-examples.mdx";
+
+The "front door" pattern involves using a Cloud Endpoint, the forward-internal Traffic Policy action, and an internal Agent Endpoint to route traffic from the public internet to your service(s).
+
+With this pattern, you can:
+
+- Host any number of services under a single hostname (e.g. `your-company.com`).
+- Decouple gateway configuration from per-service traffic management policies.
+- Prevent any driect exposure of your services or their host systems to the public internet.
```

Comment:
```suggestion
- Prevent your services and their host systems from being exposed to the public internet.
```
Rewording this to put the subject at the start of the sentence (a little easier to parse IMO) and making it a little more succinct